teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] According to the requirement, you should add the teacher class that Inherits from the Person, with the parameter `age, specialization, name = "Unknown", parent_permission = true` that is:
````ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
Thanks, @roshan-bajgain! Great catch and great lessons!